docs/testcases.md

```diff
@@ -194,13 +201,13 @@ Click on another failure line in the failed step.
 
 **Expected**: The log should scroll to that failure line.
 
-Click on 'show successful steps'.
+Click on 'show successful steps'. (DELETE ME? I can't find this on a failing mochitest log)
 
-**Expected**: Green successful step bars should appear in the top right panel.
+**Expected**: Green successful step bars should appear in the top right panel. (DELETE ME? I can't find this on a failing mochitest log)
 
-Click on a successful step.
+Click on a successful step. (DELETE ME? I can't find this on a failing mochitest log)
 
-**Expected**: The log contents should scroll to the -- Start -- line for that step.
+**Expected**: The log contents should scroll to the -- Start -- line for that step. (DELETE ME? I can't find this on a failing mochitest log)
```

Comment:
Yeah, you can remove both of these.  The scrolling should still happen with failure lines, however.